Max theme design

maxburnside opened the pull request:
This is ready for review. There might be some old and new mixed stylings

Commit:
Began editing global colors and started navbar re-styling

Commit:
Began editing form input and well container styling

Commit:
More form input styling

Commit:
Updated brand success and danger colors

Commit:
Starting on monospace styling

Commit:
Completed more navbar styling

Commit:
Removed border-bottom stylings

Commit:
Fixed input focus font-weights and color

Commit:
Font-sizing and legend styling fixes

Commit:
Updated button default styling and hover states

Commit:
Tables!

Commit:
Fixed sorta the styling of many form elements

ianks committed:
Fix some env issues

ianks committed:
Consolidate _types

ianks committed:
Some more small tweaks to _types

ianks committed:
Include our variables

ianks committed:
Consolidate navs

ianks committed:
Consolidate labels

ianks committed:
Consolidate wells

ianks committed:
Remove unused overriding/custom.scss

ianks committed:
v2.0.0

joehawker commented:
CircleCI dropped off a coverage report for [build 10292](https://circleci.com/gh/adHawk/slingshot/10292). [Click here to check it out.*](https://10292-50375431-gh.circle-artifacts.com/0/coverage/Chrome%2062.0.3202%20%28Linux%200.0.0%29/index.html)

ianks committed:
Remove buttons

ianks committed:
Do not bork on syntax errors

ianks committed:
Make panel look closer to designs

ianks committed:
v2.0.2

ianks committed:
Further refinements of panel

ianks committed:
Make navbar-inverse mimic designs

ianks committed:
v2.0.3

joehawker commented:
CircleCI dropped off a coverage report for [build 13416](https://circleci.com/gh/adHawk/slingshot/13416). [Click here to check it out.*](https://13416-50375431-gh.circle-artifacts.com/0/coverage/Chrome%2062.0.3202%20%28Linux%200.0.0%29/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13419](https://circleci.com/gh/adHawk/slingshot/13419). [Click here to check it out.*](https://13419-50375431-gh.circle-artifacts.com/0/coverage/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13423](https://circleci.com/gh/adHawk/slingshot/13423). [Click here to check it out.*](https://13423-50375431-gh.circle-artifacts.com/0/coverage/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13495](https://circleci.com/gh/adHawk/slingshot/13495). [Click here to check it out.*](https://13495-50375431-gh.circle-artifacts.com/0/coverage/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13500](https://circleci.com/gh/adHawk/slingshot/13500). [Click here to check it out.*](https://13500-50375431-gh.circle-artifacts.com/0/coverage/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13506](https://circleci.com/gh/adHawk/slingshot/13506). [Click here to check it out.*](https://13506-50375431-gh.circle-artifacts.com/0/coverage/Chrome%2062.0.3202%20%28Linux%200.0.0%29/index.html)

ianks committed:
Fix input validation states

ianks committed:
v2.0.4

joehawker commented:
CircleCI dropped off a coverage report for [build 13517](https://circleci.com/gh/adHawk/slingshot/13517). [Click here to check it out.*](https://13517-50375431-gh.circle-artifacts.com/0/coverage/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13516](https://circleci.com/gh/adHawk/slingshot/13516). [Click here to check it out.*](https://13516-50375431-gh.circle-artifacts.com/0/coverage/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13523](https://circleci.com/gh/adHawk/slingshot/13523). [Click here to check it out.*](https://13523-50375431-gh.circle-artifacts.com/0/coverage/Chrome%2062.0.3202%20%28Linux%200.0.0%29/index.html)

joehawker commented:
CircleCI dropped off a coverage report for [build 13768](https://circleci.com/gh/adHawk/slingshot/13768). [Click here to check it out.*](https://13768-50375431-gh.circle-artifacts.com/0/coverage/Chrome%2062.0.3202%20%28Linux%200.0.0%29/index.html)

ianks committed:
Fix style on control labels

Commit:
fixed multi-select button group and panel heading padding and also updated theme preview html doc

Commit:
fixed multi-select button spacing

ianks committed:
v2.0.5

Commit:
Panel title styling (#11)

Add arrows for panel titles and ensure that the finger pointer appears on hover.

Commit:
remove drop shaddows and fix form and button colors (#12)

* remove drop shaddows and fix form and button colors

* remove box-shadow from buttons

* v2.0.7

Commit:
Remove underline from inverse navbar (#13)

* remove drop shaddows and fix form and button colors

* remove box-shadow from buttons

* v2.0.7

* remove underline from inverse navbar

Commit:
v2.0.8

ianks committed:
Make table style look more like designs
